src/components/icon/assets/accessibility.js

```diff
@@ -10,18 +10,16 @@
  */
 
 import * as React from 'react';
-
 const OuiIconAccessibility = ({ title, titleId, ...props }) => (
   <svg
+    xmlns="http://www.w3.org/2000/svg"
     width={16}
     height={16}
     viewBox="0 0 16 16"
-    xmlns="http://www.w3.org/2000/svg"
     aria-labelledby={titleId}
     {...props}>
     {title ? <title id={titleId}>{title}</title> : null}
-    <path d="M8 0a8 8 0 110 16A8 8 0 018 0zm0 1a7 7 0 100 14A7 7 0 008 1zm3.974 4.342a.5.5 0 01-.233.596l-.083.036L9 6.86v2.559l.974 2.923a.5.5 0 01-.233.596l-.083.036a.5.5 0 01-.596-.233l-.036-.083-1-3L8 9.5l-.026.158-1 3a.5.5 0 01-.97-.228l.022-.088L7 9.416V6.86l-2.658-.886a.5.5 0 01.228-.97l.088.022L7.583 6h.833l2.926-.974a.5.5 0 01.632.316zM8 3a1 1 0 110 2 1 1 0 010-2z" />
+    <path d="M8 0a8 8 0 1 1 0 16A8 8 0 0 1 8 0Zm0 1a7 7 0 1 0 0 14A7 7 0 0 0 8 1Zm3.974 4.342a.5.5 0 0 1-.233.596l-.083.036L9 6.86v2.559l.974 2.923a.5.5 0 0 1-.233.596l-.083.036a.5.5 0 0 1-.596-.233l-.036-.083-1-3L8 9.5l-.026.158-1 3a.5.5 0 0 1-.97-.228l.022-.088L7 9.416V6.86l-2.658-.886a.5.5 0 0 1 .228-.97l.088.022L7.583 6h.833l2.926-.974a.5.5 0 0 1 .632.316ZM8 3a1 1 0 1 1 0 2 1 1 0 0 1 0-2Z" />
```

Comment:
Is there any specific reason that these 2 paths don't have the same definition? It seems that the old one was minified somehow. Did the older `svgr` minify the path, is the new version expanding it, etc?

Reply:
I've tried looking into it, but it lies pretty deep and I couldn't find the exact location of the change. 

`@svgr/plugin-svgo@7.0.0` now uses `svgo@3.0.2` under the hood, while `@svgr/plugin-svgo@4.0.3` used `svgo@1.1.1`. Some changes to `default plugins` were applied after `svgo@2.x.x`, so I assume the truth lies there somewhere in the settings. Unfortunately, their changelog doesn't mention anything about it, but I believe that these changes will not affect the end user in any way.